src/redux/books/books.js

```diff
@@ -0,0 +1,40 @@
+// Actions
+const ADD = 'ADD';
+const REMOVE = 'REMOVE';
+
```

Comment:
- [ ] Kindly create your actions using the _**duck pattern**_. It should look like this:

```
const ADD = 'bookstore/books/ADD'
```

#### Kindly implement this change. You can read this [article](https://github.com/erikras/ducks-modular-redux) to get a better understanding of duck patterns.